### PR TITLE
Add a side bar to admin pages.

### PR DESCRIPTION
**Pull request by gimite**
#121

This provides consistent way to navigate between admin pages.

s6yjixv5eqo

### CODE REVIEW

File: app/admin_resources.py [hunk 1]
```diff
@@ -139,10 +139,12 @@ def format_nav_html(self, bundle_name, name, lang):
...
             crumbs.append((anchor, (bundle_name, name, lang)))
         last = crumbs[-1][1]
-        return '<div class="nav">%s</div>' % (' &gt; '.join(
-            '<a class="%s" href="%s">%s</a>' %
-            (args == last and 'sel', self.get_admin_url(*args), html(anchor))
-            for anchor, args in crumbs))
+        anchors = (
+            ['<a href="%s">Admin page</a>' % self.get_url('admin')] +
+            ['<a class="%s" href="%s">%s</a>' %
+                (args == last and 'sel', self.get_admin_url(*args), html(anchor))
```
Comment (kevinalle) [inline app/admin_resources.py]: args == last and 'sel'
might set class="False" on the element, no?

File: app/main.py [hunk 2]
```diff
@@ -38,6 +39,35 @@
...
+        try:
+            return [
+                utils.Struct(
+                    repo=repo,
+                    url=utils.get_repo_url(self.request, repo) + '/admin')
+                for repo in sorted(model.Repo.list())]
+        except:
+            # Logs the exception here because exceptions thrown during template
+            # variable evaluation is silently ignored.
+            logging.exception('exception')
```
Comment (kevinalle) [inline app/main.py]: Are you sure you want to log the string 'exception'? and not the actual exception?

**Comment by kevinalle** [comment]
Please see comments. Otherwise LGTM!

File: app/admin_resources.py (hunk 1 above)
Comment (gimite) [inline app/admin_resources.py]: Good catch, though it was a bug before my change :) Fixed.

File: app/main.py (hunk 2 above)
Comment (gimite) [inline app/main.py]: logging.exception() logs the current exception by default. Added a comment.

**Comment by gimite** [comment]
Thanks! Merged.

I also needed to fix some tests.

**Pull request merged**